modules/3d-tiles/src/tiles-3d-loader.ts

```diff
@@ -29,7 +29,7 @@ export type Tiles3DLoaderOptions = LoaderOptions &
 /**
  * Loader for 3D Tiles
  */
-export const Tiles3DLoader: LoaderWithParser = {
+export const Tiles3DLoader: LoaderWithParser<unknown, never, Tiles3DLoaderOptions> = {
```

Comment:
- Typing the returned value from this loader is the single biggest service we can do to the user. Returning `any` or `unknown` is really not helping anyone.
- I would say that this is a P0 for loaders.gl v4